Update AppGW SSL Policy notes and process

jolleymd opened the pull request:
Need to make sure users know that the AppGW SSL Policy cmdlets have parameters that cannot mix based on the PolicyType specified. Also added section for updating an AppGW since users tend to miss the Set for the AppGw, itself.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cjgronlund commented:
@ClarityQueen 
Deprecated content that needs to be removed from ACOM.

jolleymd committed:
Update AppGW SSL Policy notes and process

Need to make sure users know that the AppGW SSL Policy cmdlets have parameters that cannot mix based on the PolicyType specified. Also added section for updating an AppGW since users tend to miss the Set for the AppGw, itself.
